config/bbp_maintainers_config.nix

```diff
@@ -8,6 +8,7 @@
     FrancescoCasalegno = "Casalegno Francesco <francesco.casalegno@epfl.ch>";
     jamesgkind = "James Gonzalo King <jamesgonzalo.king@epfl.ch>";
     jplanasc = "Judit Planas <judit.planas@epfl.ch>";
+    matze = "Matthias Wolf <matthias.wolf@epfl.ch>";
```

Comment:
You should add yourself to the maintainers list of the recipe you added

Reply:
Done.